packages/core/src/card/card.stories.ts

```diff
@@ -249,39 +239,67 @@ export function WithImage() {
   </cds-card>`;
 }
 
-export function WithLayoutAndOverflow() {
-  return html` <style>
-      cds-card#cds-card-with-fixed-height #cds-card-with-fixed-height__body {
-        height: 1rem;
-        overflow: auto;
-      }
-    </style>
-    <cds-card id="cds-card-with-fixed-height">
-      <div cds-layout="vertical gap:md">
-        <div cds-text="section" cds-layout="p-y:sm">
-          Card Title
-        </div>
-
-        <cds-divider cds-card-remove-margin></cds-divider>
-
+export function SocialPost() {
+  ClarityIcons.addIcons([
+    'batman',
+    renderIcon(`
+    <svg viewBox="29.25 -183.153 985.166 985.166" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Running the svg through https://jakearchibald.github.io/svgomg/  will save up some space

Reply:
from `9.1k` to `3.7k`. Thanks!